business/test_util.go

```diff
@@ -577,6 +578,44 @@ func FakePodsSyncedWithDeployments() []v1.Pod {
 	}
 }
 
+func FakePodSyncedWithDeployments() *v1.Pod {
```

Comment:
That could go as well in `kubernetes/kubetest/mock.go`, closer to the k8s layer (anyway I guess I will create a new task to consolidate all our k8s resources mocks into a single package)

Reply:
I admit to just inserting the necessary additions into the existing code layout.  I'm certainly not against a restructuring.